packages/variants/src/variants.ts

```diff
@@ -15,6 +16,17 @@ function makeThemeFromPartials(
   });
 }
 
+function changeOpacity(inputColor: string, opacity: number): string {
+  let newColor: IColor | undefined = getColorFromString(inputColor);
+  if (!newColor) {
+    return inputColor;
+  }
+  newColor = updateA(newColor, opacity);
```

Comment:
so just change this line to `return updateA(newColor, opacity).str;` and then delete the rest of the method

Reply:
Updated and tested. Also added function description